packages/NodeTypeResolver/src/DependencyInjection/PHPStanServicesFactory.php

```diff
@@ -20,15 +20,22 @@ final class PHPStanServicesFactory
 
     public function __construct()
     {
-        $containerFactory = new ContainerFactory(getcwd());
+        $currentWorkingDirectory = getcwd();
+
+        $containerFactory = new ContainerFactory($currentWorkingDirectory);
         $additionalConfigFiles = [];
 
         // possible path collision for Docker
-        $phpstanPhpunitExtensionConfig = getcwd() . '/vendor/phpstan/phpstan-phpunit/extension.neon';
+        $phpstanPhpunitExtensionConfig = $currentWorkingDirectory . '/vendor/phpstan/phpstan-phpunit/extension.neon';
         if (file_exists($phpstanPhpunitExtensionConfig) && class_exists('PHPUnit\\Framework\\TestCase')) {
             $additionalConfigFiles[] = $phpstanPhpunitExtensionConfig;
         }
 
+        $currentProjectConfigFile = $currentWorkingDirectory . 'phpstan.neon';
```

Comment:
Maybe?

```suggestion
        $currentProjectConfigFile = $currentWorkingDirectory . '/phpstan.neon';
```

Reply:
Not maybe. Definitely.

Reply:
Great!

The older I am, the more I trust passing test over my guess :D